specification/mediaservices/resource-manager/Microsoft.Media/preview/2018-03-30-preview/ContentKeyPolicies.json

```diff
@@ -0,0 +1,1167 @@
+            "description": "Specifies a non-negative integer n that limits the number of items returned from a collection. The service returns the number of available items up to but not greater than the specified value n."
+          },
+          {
+            "name": "$orderby",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Specifies the the key by which the result collection should be ordered."
+          }
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "@odata.nextLink"
+        },
+        "x-ms-odata": "#/definitions/ContentKeyPolicy",
+        "x-ms-examples": {
+          "Lists all Content Key Policies": {
+            "$ref": "examples/content-key-policies-list-all.json"
+          },
+          "Lists Content Key Policies ordered by last modifed.": {
+            "$ref": "examples/content-key-policies-list-by-lastModified.json"
+          },
+          "Lists Content Key Policies with created and last modified filters.": {
+            "$ref": "examples/content-key-policies-list-in-date-range.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/contentKeyPolicies/{contentKeyPolicyName}": {
+      "get": {
+        "summary": "Get a Content Key Policy",
+        "description": "Get the details of a Content Key Policy in the Media Services account",
+        "operationId": "ContentKeyPolicies_Get",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ContentKeyPolicy"
+            }
+          },
+          "404": {
+            "description": "NotFound"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "contentKeyPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Content Key Policy name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-examples": {
+          "Get a Content Key Policy by name": {
+            "$ref": "examples/content-key-policies-get-by-name.json"
+          }
+        }
+      },
+      "put": {
+        "summary": "Create or update an Content Key Policy",
+        "description": "Create or update a Content Key Policy in the Media Services account",
+        "operationId": "ContentKeyPolicies_CreateOrUpdate",
```

Comment:
>ContentKeyPolicies_CreateOrUpdate [](start = 24, length = 33)

Is this a long running operation?

Reply:
No, the service just stores the record internally.